template/web/app/global.css

```diff
@@ -1,3 +1,6 @@
+/* Google Fonts - Roboto Mono */
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@0,100..700;1,100..700&display=swap');
```

Comment:
Why this?

I thought we already have this font download.

Reply:
@Zizzamia oh do we? I didn't see it in the codebase.

Reply:
Yes you're right, I'll fix